plugin-utilities-test/src/main/kotlin/org/jetbrains/research/pluginUtilities/util/FileTestUtil.kt

```diff
@@ -85,3 +85,5 @@ object FileTestUtil {
 fun getPsiFile(file: File, fixture: CodeInsightTestFixture): PsiFile {
     return fixture.configureByFile(file.path)
 }
+
+fun noInputError(): Nothing = error("Parametrized test did not receive input")
```

Comment:
Since it is a general function, please use an argument with a subject that did not receive input and use it on the error message